api/services/User/index.js

```diff
@@ -63,34 +63,22 @@ export default class User extends BaseModelService {
         ],
       });
     });
-    return Promise.all(resultPromise);
+    const result = await Promise.all(resultPromises);
+    return result.map((data) => !data);
   }
 
-  static availabilityLoader = new DataLoader(
-    this.batchFunctionForAvailabilityLoader,
-  );
-
-  static async findAvailaibilityDuring(
-    user_id,
-    dateTimeRange,
-    exculdeEvents = [],
-  ) {
-    const keyForDataLoader = {
-      user_id,
-      dateTimeRange,
-      exculdeEvents,
-    };
-    const data = await this.availabilityLoader.load(keyForDataLoader);
-    return !data;
+  static getAvailabilityLoader() {
+    return new DataLoader(this.findAllAvailabilityDuring);
```

Comment:
OOPS 101: can't use `this` inside static methods. It should be `User.findAllAvailabilityDuring`.